### PR TITLE
Upgrade to `credo-ts` v0.5.2

### PR DESCRIPTION
**Pull request by jonmattgray**
Upgrade Credo dependencies to `0.5.2` https://github.com/openwallet-foundation/credo-ts/releases/tag/v0.5.2
Includes rename from `aries-framework` to `credo-ts`

### CODE REVIEW

File: src/controllers/outofband/OutOfBandController.ts [hunk 1]
```diff
@@ -140,7 +140,7 @@ export class OutOfBandController extends Controller {
   @Example<{ message: AgentMessageType; invitationUrl: string }>({
     message: {
       '@id': 'eac4ff4e-b4fb-4c1d-aef3-b29c89d1cc00',
-      '@type': 'https://didcomm.org/connections/1.0/invitation',
+      '@type': 'https://didcomm.org/connections/1.x/invitation',
```
Comment (jonmattgray) [inline src/controllers/outofband/OutOfBandController.ts:143]: https://github.com/openwallet-foundation/credo-ts/blob/main/packages/core/src/modules/connections/models/HandshakeProtocol.ts

File: src/controllers/examples.ts [hunk 2]
```diff
@@ -205,18 +208,13 @@ export const outOfBandRecordExample = {
...
     threadId: '82701488-b43c-4d7b-9244-4bb204a7ae26',
     connectionId: 'ac6d0fdd-0db8-4f52-8a3d-de7ff8ddc14b',
   },
-  metadata: {
-    '_internal/indyCredential': {
-      credentialDefinitionId: 'q7ATwTYbQDgiigVijUAej:3:CL:318187:latest',
-      schemaId: 'q7ATwTYbQDgiigVijUAej:2:Employee Badge:1.0',
-    },
-  },
+  role: CredentialRole.Issuer,
```
Comment (jonmattgray) [inline src/controllers/examples.ts:217]: `role` is now required https://github.com/openwallet-foundation/credo-ts/blob/e58ec5bd97043d57fcc3c5a4aee926943e6c5326/packages/core/src/modules/credentials/repository/CredentialExchangeRecord.ts#L19

File: src/controllers/examples.ts [hunk 3]
```diff
@@ -230,6 +228,7 @@ export const CredentialExchangeRecordExample = {
 export const ProofRecordExample: ProofExchangeRecordProps = {
   id: '821f9b26-ad04-4f56-89b6-e2ef9c72b36e',
   protocolVersion: 'v2',
+  role: ProofRole.Verifier,
```
Comment (jonmattgray) [inline src/controllers/examples.ts:231]: `role` is now required https://github.com/openwallet-foundation/credo-ts/blob/e58ec5bd97043d57fcc3c5a4aee926943e6c5326/packages/core/src/modules/proofs/repository/ProofExchangeRecord.ts#L15

**Review by Ellenn-A — APPROVED** [review]
seems reasonable, ty for linking stuff to have a look  at/note

**Review by msterle — APPROVED** [review]
lgtm!

**Pull request merged**